crates/oxc_cli/src/lib.rs

```diff
@@ -38,7 +38,25 @@ impl Cli {
             .cli_options
             .paths
             .iter()
-            .flat_map(|path| Walk::new(path).iter().collect::<Vec<_>>())
+            .flat_map(|path| {
+                Walk::new(path)
+                    .iter()
+                    .filter(|path| {
+                        let ignore_pattern = &self.cli_options.ignore_pattern;
+                        for pattern in ignore_pattern {
+                            if pattern.to_str() == path.to_str() {
+                                return false;
+                            }
+
+                            if pattern.is_dir() && path.starts_with(pattern) {
+                                return false;
+                            }
+                        }
```

Comment:
pattern can be a glob, `path.starts_with(pattern)` is just part of it.

> Argument Type: String. The supported syntax is the same as for [.eslintignore files](https://eslint.org/docs/latest/use/configure/ignore#the-eslintignore-file), which use the same patterns as the [.gitignore specification](https://git-scm.com/docs/gitignore). You should quote your patterns in order to avoid shell interpretation of glob patterns.

Reply:
I think [this](https://github.com/Boshen/oxc/pull/76/files#diff-2e99f30fd4b212435f2dfa85c862bd8a0c3f30a062fb35862ae7f49749c310a0R38-R48) handle the glob pattern, doesn't it?